src/main/webapp/app/views/admin/settings/application/lookAndFeel.html

```diff
@@ -3,15 +3,51 @@
 		<div class="row">
 			<div class="col-sm-6 col-lg-9">
 
-				<div class="row">
-					<h4>Background
-					   <div tooltip="Use hex color values to change the background on the student side. The main color (default #1B333F) fades into the highlight color (default #43606E).">
-					   	<span class="glyphicon glyphicon-info-sign opaque color-reset"></span>
-					   </div>
-					</h4>
+				<div class="row look-and-feel-h1">
+					<div>Text
+					   <span class="glyphicon glyphicon-info-sign opaque color-reset" tooltip="Use hex color values to change the default text color for the application."></span>
+					</div>
 				</div>
 
-				<div class="row">
+				<div class="row look-and-feel-h2">
+					<div class="col-md-5 col-lg-3">
+						<div class="row">
+							<span>Main Color
+								<span
+									class="glyphicon glyphicon-refresh color-reset"
+									ng-click="resetConfiguration('lookAndFeel','text_main_color')">
+								</span>
+							</span>
+						</div>
+
+						<div class="input-group color-selector">
+							<span class="input-group-addon">#</span>
+						  	<input
+						  		class="form-control color-input"
+						  		ng-change="updateHexValue('lookAndFeel','text_main_color')"
+						  		ng-model="settings.configurable.lookAndFeel.text_main_color.value"
+						  		dehashcolor>
+
+							<span class="input-group-addon">
+								<label class="color-selector-swatch" ng-style="{background: settings.configurable.lookAndFeel.text_main_color.value}">
```

Comment:
Would it be better to use a custom class and have that class setting the background added here rather than using `ng-style`?

Reply:
Okay, so this is existing design choice behavior.